.github/workflows/continous-integration.yml

```diff
@@ -291,9 +293,11 @@ jobs:
 
       - name: Install Dependencies (Linux) 📦
         if: needs.changes.outputs.backend == 'true' && matrix.os == 'ubuntu-22.04'
+        # Poetry intermittently fails to install dependency if it is not PEP 517 compliant
+        # This is a workaround for that issue
         run: |
           sudo apt-get -y install libpq-dev
-          make install-full | tee .output
+          make install-full | tee .output  || make install-full | tee .output || make install-full | tee .output
```

Comment:
@sanchariGr can you maybe validate if this is the right approach to apply temporary "fix" the dependency installation?

Reply:
@radovanZRasa I think this seems to be the best way forward for now.